backend/routes/notice.js

```diff
@@ -0,0 +1,43 @@
+var express = require("express");
+var router = express.Router();
+
+const Notice = require("../models/Notice.model");
+
+router.post("/", function (req, res, next) {
+  const notice = new Notice({
+    title: req.body.title,
+    description: req.body.description,
+    pdfPath: req.body.pdfPath || null,
+    eventLink: req.body.eventLink || null,
+  });
+
+  notice
+    .save()
+    .then((result) => {
+      res.status(201).json({
+        result: result,
+      });
+      console.log(result);
+    })
+    .catch((err) => {
+      res.status(500);
+      console.log(err);
+    });
+});
+
+// Returns all the Notices (sorted : latest first)
+router.get("/all", (req, res, next) => {
```

Comment:
We can have `router.get("/", ...` to get all the notices instead of `"/all"`. Just a suggestion.

Reply:
the reason for /all is, route name is '/notice' and not '/notices'
so either we can make /notices or /notice/all
let me know what dou you think

Reply:
The thing is that we'll be having client-side routing with react-router-dom so, having descriptive API fetch-points will be good. Its all green for me. 👍🏻